problem_builder/mentoring.py

```diff
@@ -284,21 +285,26 @@ def student_view(self, context):
         fragment = Fragment()
         child_content = u""
 
+        activate_child_id = context.get('activate_child_id', None)
```

Comment:
+1, nice approach. I like the fact that this would allow other blocks to alter their presentation based on the selected block, even if it's not the current block.